meteor/server/api/ingest/rundownInput.ts

```diff
@@ -1170,9 +1174,8 @@ function updateSegmentFromIngestData(
 	playlist: RundownPlaylist,
 	rundown: Rundown,
 	ingestSegment: IngestSegment
-): SegmentId | null {
+): { segmentId: SegmentId | null; newPartIds: string[] } {
```

Comment:
It would be better to call these `insertedPartExternalIds`, as they are externalIds

Reply:
Renamed as requested.